lib/config/presets/internal/replacements.ts

```diff
@@ -99,6 +121,114 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'material-ui/icons-to-mui/icons-material': {
+    description: 'the @material-ui/ monorepo org was renamed to @mui/',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=4.0.0 <5.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/icons'],
+        replacementName: '@mui/icons-material',
+        replacementVersion: '5.0.0',
+      },
+    ],
+  },
+  'material-ui/lab-to-mui/lab': {
+    description: 'the @material-ui/ monorepo org was renamed to @mui/',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=4.0.0 <5.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/lab'],
+        replacementName: '@mui/lab ',
+        replacementVersion: '5.0.0',
+      },
+    ],
+  },
+  'material-ui/private-theming-to-mui/private-theming': {
+    description: 'the @material-ui/ monorepo org was renamed to @mui/',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=4.0.0 <5.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/private-theming'],
+        replacementName: '@mui/private-theming',
+        replacementVersion: '5.0.0',
+      },
+    ],
+  },
+  'material-ui/styled-engine-sc-to-mui/styled-engine-sc': {
+    description: 'the @material-ui/ monorepo org was renamed to @mui/',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=4.0.0 <5.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/styled-engine-sc'],
+        replacementName: '@mui/styled-engine-sc',
+        replacementVersion: '5.0.0',
+      },
+    ],
+  },
+  'material-ui/styled-engine-to-mui/styled-engine': {
+    description: 'the @material-ui/ monorepo org was renamed to @mui/',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=4.0.0 <5.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/styled-engine'],
+        replacementName: '@mui/styled-engine',
+        replacementVersion: '5.0.0',
+      },
+    ],
+  },
+  'material-ui/styles-to-mui/styles': {
+    description: 'the @material-ui/ monorepo org was renamed to @mui/',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=4.0.0 <5.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/styles'],
+        replacementName: '@mui/styles',
+        replacementVersion: '5.0.0',
+      },
+    ],
+  },
+  'material-ui/system-to-mui/system': {
+    description: 'the @material-ui/ monorepo org was renamed to @mui/',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=4.0.0 <5.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@material-ui/system'],
+        replacementName: '@mui/system',
+        replacementVersion: '5.0.0',
+      },
+    ],
+  },
+  'material-ui/types-ui-to-mui/types': {
```

Comment:
It was previously called `@material-ui/types`, so I don't think the "ui-" before "to" is right.